test_stack.py

```diff
@@ -1,27 +1,33 @@
-import pytest
+import io, pytest, sys
 
-from stack import Stack
+from stack import Stack, LinkedList
```

Comment:
Why are you importing LinkedList here?

Reply:
i had that there because i was messing around with different ways to write a test. But I don't use it so will remove it